### PR TITLE
CART-977 tests: notify of memcheck errors

### PR DESCRIPTION
**Pull request by mallove79**

### CODE REVIEW

File: src/tests/ftest/cart/util/cart_utils.py [hunk 1]
```diff
@@ -418,6 +418,52 @@ def build_cmd(self, env, host, **kwargs):
...
+                ", ".join(memcheck_files))
+
+        saved_cwd = os.getcwd()
+        os.chdir(daos_test_shared_dir)
+
+        # Rename the file so it's not checked again for memcheck errors
+        for filename in memcheck_files:
+            os.rename(filename, filename + "-checked")
+
+        os.chdir(saved_cwd)
```
Comment (ashleypittman) [inline src/tests/ftest/cart/util/cart_utils.py:463]: You shouldn't be using chdir here
```suggestion
        # Rename the file so it's not checked again for memcheck errors
        for filename in memcheck_files:
            log_file = os.path.join(daos_test_shared_dir, filename)
            os.rename(log_file, log_file + "-checked")
```

**Review by ashleypittman — COMMENTED** [review]
What effect does this have overall, is it just putting an entry in the logs when there are errors?

Ultimately we want the xml files (and results) to be reported and rendered through Jenkins itself don't we, and that depends on the plugin being configured to pick up the right filenames, this PR will leave them with a "-checked" suffix which almost certainly doesn't match that.

**Comment by mallove79** [comment]
> What effect does this have overall, is it just putting an entry in the logs when there are errors?

Correct, we currently don't display any Valgrind memcheck results for cart functional tests.

> Ultimately we want the xml files (and results) to be reported and rendered through Jenkins itself don't we, and that depends on the plugin being configured to pick up the right filenames, this PR will leave them with a "-checked" suffix which almost certainly doesn't match that.

Correct -- this PR is a stop gap between now and integrating with the Jenkins Valgrind plugin ([CART-972](https://daosio.atlassian.net/browse/CART-972)), which will revert these XML file name changes.

**Comment by sylviachanoiyee** [comment]
@mallove79 , is this PR still relevant/needed? Last activity was 2 months ago but gatekeeping was not requested.

**Comment by mallove79** [comment]
> @mallove79 , is this PR still relevant/needed? Last activity was 2 months ago but gatekeeping was not requested.

No longer relevant.  https://github.com/daos-stack/daos/pull/6945 should replace this PR.

**Comment by sylviachanoiyee** [comment]
Maybe close the PR is not relevant?

**Comment by mallove79** [comment]
#6945 replaces this PR.

**Pull request closed without merging**